### ISSUE TITLE
add LMS and DKL color space colorschemes?

### ISSUE THREAD

**Issue by babaq** [JuliaGraphics/ColorSchemes.jl#55]
Hi,

I have generated and used some basic colormaps for color science studies. These include:

LMS Cone Isolating colors
![image](https://user-images.githubusercontent.com/92771/102279252-ed9c3a80-3edf-11eb-9b84-095de0684c02.png)

DKL Cone-Opponent colors
![image](https://user-images.githubusercontent.com/92771/102279335-0f95bd00-3ee0-11eb-9b37-16ce79bc5b16.png)

DKL Isoluminance Hues
![image](https://user-images.githubusercontent.com/92771/102279601-85018d80-3ee0-11eb-8958-3619711427f2.png)

and other variants.

I am wandering if these colormaps are of more general interests, so anyone can easily get access to them if they were included in the predefined colorschemes.

**Comment by cormullion** [comment]
Sounds like a good idea! 

Most of the predefined colorschemes here are"documented" elsewhere, with a link to the originating site (see https://juliagraphics.github.io/ColorSchemes.jl/latest/basics/). And there are usually license files to go with them, in /data. So you could explain and document them somewhere (eg your web site) and "publish" them, then it's easy to include them here with a suitable link to "the source".

**Comment by cormullion** [comment]
Did you get round to writing these up somewhere?

**Comment by babaq** [comment]
i have these scattered in several packages, do you suggest create a new small pacakge only for them, or i could move them all into one packages(the [ColorLab](https://github.com/Experica/ColorLab.jl))?

**Comment by cormullion** [comment]
Well it's just that there's not enough space to describe each colour scheme in any detail in the docs, so I like to just add a link to a place where the purpose and nature of the schemes is provided in more detail. Eg for `cmocean` I can just link to their site (https://juliagraphics.github.io/ColorSchemes.jl/latest/basics/#cmocean) where they have all the credits, etc.. 

To be honest I suppose it could just be a README.md or a section in any documentation you might already have. :)

**Comment by babaq** [comment]
I see your point. what about the scheme data, right now I save them in a JLD2 file, and loaded in package __init__(), but I remember some of the concreate colors in Colors.jl are written down directly in the source file, is this the way of colorschemes?

**Comment by cormullion** [comment]
Yes, every colorscheme in this package is simply an array of RGB{Float64}.  The idea of the package is simply to act as a simple catalogue of pre-defined colorschemes - unlike some other packages where colorschemes are generated when required.
